openhtf/io/proto/test_runs.proto

```diff
@@ -17,9 +17,41 @@ syntax = "proto2";
 
 package openhtf;
 
+import "assembly.proto";
```

Comment:
If this line came from a google3 version of the file and you want to keep the diff as minimal as possible, you can comment it out rather than remove it (and other references to assembly stuff).